dummy_data_generation/generate_data.py

```diff
@@ -71,6 +72,20 @@ def generate_survey_v2_data(directory, file_date, records, swab_barcodes, blood_
     return survey_responses
 
 
+def generate_digital_data(directory, file_date, records, swab_barcodes, blood_barcodes):
+    """
+    Generate survey digital data.
+    """
+    schema = Schema(
+        schema=get_survey_responses_digital_data_description(
+            _, swab_barcodes=swab_barcodes, blood_barcodes=blood_barcodes
+        )
+    )
+    survey_responses = pd.DataFrame(schema.create(iterations=records))
+    write_output(survey_responses, directory / f"ONSE_CISDigital_v1_0_responses_{file_date}.txt", "|")
```

Comment:
Add the time element of the datetime for digital
```suggestion
    write_output(survey_responses, directory / f"ONSE_CISDigital_v1_0_responses_{file_date}_000000.txt", "|")
```